src/components/domain/post/blog/clanky-constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ITEMS_PER_PAGE = 9
 export const CATEGORY_NAME = 'Blog'
 export const EMPTY_BLOG_IMAGE_PLACEHOLDER_URL =
   'http://wp.streetofcode.sk/wp-content/uploads/2022/09/Group-141.png'
+export const RECOMMENDED_POSTS_NUMBER = 5
```

Comment:
```suggestion
export const RECOMMENDED_POSTS_COUNT = 5
```

Oh and perhaps we could change it either into 3 or 6 so that they are nicely aligned. I think 3 should be enough, but I'm good with 6 as well.

Reply:
I'll change it to 3 so it looks better and also number of requests will be lower in average for each post.